other/materials_designer/create_nanowire.ipynb

```diff
@@ -0,0 +1,242 @@
+{
```

Comment:
<p>Separate the part where the wire is rotated into its own cell.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Exabyte-io/api-examples/pull/162/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>